### DISCUSSION TITLE
Icon Edit

### DISCUSSION THREAD

**Discussion by xe-kohler** [mckaywrigley/chatbot-ui#1462] [Help]
Hey team! 
This may sound like a really silly question; how do you change the icons in the project? I am trying to customise the UI and really struggling. I've changed the icon files in the project, given them the same name as previous, and hunted through the project files to see where they're called but I just cannot seem to find them / figure out how they're being called. Please help! 
Thank you!!

**Comment by zshmh** [comment]
I'm also facing this problem, did you solve it?

**Comment by xe-kohler** [comment]
Yeah I did (apologies forgot I posted here). If I’m remembering correctly: Find the brand file, and you should see the icon in the form of a coded svg. You’ll need to gather your new icon as svg code and replace it inside of the the <svg> tags. On 23-03-2024, at 3:33 AM, zshmh ***@***.***> wrote:﻿
I'm also facing this problem, did you solve it?

**Comment by TargetIQ** [comment]
can also just replace with PNG 

import { FC } from "react";

interface ChatbotUISVGProps {
  scale?: number;
}

export const ChatbotUISVG: FC<ChatbotUISVGProps> = ({ scale = 1 }) => {
  const imageSize = { width: 112.15 * scale, height: 136.56 * scale };

  return (
    <img
      src="https://example.com/uploads/companylogo-256x256-1.png"
      alt="Logo"
      style={{ width: `${imageSize.width}px`, height: `${imageSize.height}px` }}
    />
  );
};
